### ISSUE TITLE
single-key-in linter gives exception on usage of assoc-in without args

### ISSUE THREAD

**Issue by borkdude** [borkdude/clj-kondo#845]
```
$ clj-kondo --lint - --config '{:linters {:single-key-in {:level :warning}}}' <<< '(assoc-in)'
```
Unexpected error. Please report an issue.
java.lang.IndexOutOfBoundsException
